web-components/src/components/icons/EmptyCartIcon.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { SxProps } from '@mui/material';
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+import { Theme } from '~/theme/muiTheme';
+
+interface IconProps extends SvgIconProps {
+  sx?: SxProps<Theme>;
+}
+
+export default function EmptyCartIcon({
+  sx = [],
+  ...props
+}: IconProps): JSX.Element {
+  return (
+    <SvgIcon
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 98 84"
+      sx={[...(Array.isArray(sx) ? sx : [sx])]}
+      {...props}
+    >
+      <path
+        d="M17.5 17.5h54.753a1 1 0 0 1 .977 1.217l-6.556 29.5a1 1 0 0 1-.976.783H25.302a1 1 0 0 1-.976-.783L17.5 17.5Z"
+        stroke="#8F8F8F"
+        strokeWidth={4}
+      />
+      <path
+        d="M5 3h7.915a1 1 0 0 1 .974.774l10.178 43.939a1 1 0 0 1-.127.757l-7.528 11.998A1 1 0 0 0 17.26 62H59"
+        stroke="#8F8F8F"
+        strokeWidth={4}
+      />
+      <rect
+        x={3}
+        y={5}
+        width={4}
+        height={10}
+        rx={0.25}
+        transform="rotate(-90 3 5)"
+        fill="#8F8F8F"
```

Comment:
also no need to change before merge, but it could be good to use `currentColor` here instead of a hex value